React.podspec

```diff
@@ -48,7 +48,6 @@ Pod::Spec.new do |s|
                               "IntegrationTests/*",
                               "React/DevSupport/*",
                               "React/Inspector/*",
-                              "React/**/RCTTVView.*",
```

Comment:
Can you mark the TVView files as tvOS only? I believe cocoapods supports specifying this.